### DISCUSSION TITLE
Randomizing Eggs

### DISCUSSION THREAD

**Discussion by Ninjistix** [kwsch/PKHeX#3680] [Q&A] [answered]
I wanted to Generate some shiny eggs for random pokemon
I'm trying to randomize the met date and the egg date to the same

I found this https://projectpokemon.org/home/forums/topic/56800-randomise-pkhex-metdate-in-batch-editor/

It contains 

.Met_Day=$1,28
.Met_Month=$1,12
.Egg_Day=$1,28
.Egg_Month=$1,12

I saw kwsch say to use $suggest for the egg to have it it set to the same metday but that's not working I'm getting an error

=Species=4
.Met_Day=$1,8
.Met_Month=12
.Egg_Day=$suggest
.Egg_Month=$suggest

![image](https://user-images.githubusercontent.com/12572974/206616406-f1d1f4df-3214-4a97-8206-ff4aab8c363e.png)

**Comment by sora10pls** [comment, marked as answer]
You are running `$suggest` on the wrong fields.
`.EggMetDate=$suggest`

**Reply by Ninjistix** [reply]
Would it be the same for met month?

**Reply by sora10pls** [reply]
What? Just replace the `Egg_Day` and `Egg_Month` commands in your original post with the one I shared.

**Reply by Ninjistix** [reply]
okay thanks a lot, it worked.

**Reply by Ninjistix** [reply]
Can you delete/close this?

**Reply by kwsch** [reply]
Discussions don't need to be closed. Usually the OP marks a question reply as the "answer" and the discussion doesn't have to be locked after.
